apps/cowswap-frontend/src/modules/account/containers/Transaction/ActivityDetails.tsx

```diff
@@ -300,6 +324,19 @@ export function ActivityDetails(props: {
                   </>
                 )}
               </SummaryInnerRow>
+
+              {/* TODO: Get dynamic recipient */}
+              {isCustomRecipient && (
+                <SummaryInnerRow>
+                  <b>Recipient:</b>
+                  <i>
+                    {isWarningRecipientIconVisible && <Icon image={IconType.ALERT} color={UI.COLOR_ALERT} description="Alert" />}
+                    <a href="#" target="_blank" rel="noopener nofollow">0xfE84A86803952041eCCCB584300eb66C601BfeC1 ↗</a>
```

Comment:
You can use the ExplorerLink component to use Etherscan for the address.
And the address should be available from `order.recipient`.

Reply:
Added your suggestion